Jason/Store.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+Python OOP Assignment
+
+Author: Jason Lee
+
+"""
+
+### Create class of Product
+class Product(object):
+
+	def __init__(self, price, item_name, weight, brand, cost, status= "for sale"):
+ 		self.price = price
+	  	self.item_name = item_name
+	  	self.weight = weight
+	  	self.brand = brand
+	  	self.cost = cost
+	  	self.status = status
+
+  	def sell(self):
+	  	self.Status = "sold"
+		return self
+
+  	def add_tax(self):
+  		self.tax = 0.05*self.price
+  		return self.price + self.tax
+
+  	def Return(self,reason):
```

Comment:
use a lowercase method name here. Stick to the same naming conventions throughout your code. Classes are capital case, methods are camelcase (ex. addTax), and variable names are snake case (ex. self.item_name).